airflow/settings.py

```diff
@@ -519,3 +520,13 @@ def initialize():
 # By default this is off, but is automatically configured on when running task
 # instances
 MASK_SECRETS_IN_LOGS = False
+
+# Display flash messages on the dashboard
+# Useful for warning about setup issues or announcing changes to end users
+# You can choose what role(s) the message will be shown to as well, or pass None to show to all users
+# (message, category, [role(s)]), for example:
+# ("Airflow update happening next week", "warning", ["User"])
+# You can also pass the message as flask.Markup to include HTML in the message:
+# (Markup('Visit <a href="http://airflow.apache.org">airflow.apache.org</a>'), "info", None)
+FlashMessage = Tuple[Union[str, Markup], str, Optional[str]]
```

Comment:
I considered creating a class so we can simplify the experience here a bit, e.g. something like:

```
DASHBOARD_FLASH_MESSAGES = [
    FlashMessage("Hello"),
    FlashMessage("Word", roles=["Admin"]),
    FlashMessage("example", category="Warning"),
]
```

Thoughts?

Reply:
A custom class feels like a good idea to me. We can also prevent explicitly depending on `Markup` for simple cases e.g. `FlashMessage("<a href="example.com">Example</a>", html=True)`. Although `Markup` would still be needed if the user needs more elaborated stuff like `.format()`.

Speaking of `format()`, we should add to the documentation (which I assume will come later) the implied danger using raw HTML and how to do thing property (I think MarkupSafe has some writeup on this). Maybe we should also ask Andrew how these things are handled in Django-land.

Reply:
Thanks @uranusjr, good feedback and I think I've covered it all at this point.